deploy/swrs/extract/table/eccc_attachment.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy ggircs:swrs/extract/table/eccc_attachment to pg
+-- requires: schema_ggircs_swrs
+
+begin;
+
+create table swrs_extract.eccc_attachment (
+  id integer generated always as identity primary key,
+  imported_at timestamp with time zone not null default now(),
+  attachment_file_path varchar(1000),
+  attachment_file_md5_hash varchar(1000),
+  zip_file_id integer references swrs_extract.eccc_zip_file(id) not null
+);
+
+create index eccc_attachment_zip_file_idx on swrs_extract.eccc_attachment(zip_file_id);
+
```

Comment:
This index here is just because foreign keys need to be indexed. 

I rewrote the full/incremental DAGs so that the incremental can restart where it left off if there was a failure. Might still need some tweaks.
In order to do a "full" extract, one would delete all the records from the extract schema and then run the one swrs extract dag.
I will add a DAG that does the deletion, so that we're not messing with the prod database manually